index.html

```diff
@@ -0,0 +1,232 @@
+                </p>
+                <footer>
+                   <ul id="skills_2" class="skills">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                   </ul>
+                   <div class="external_link">
+                     <a href="#">See Project</a>
+                   </div>
+                </footer>
+            </div>
+        </section>
+        <section  id="p4">
+            <img  class="project_image" src="images/Snapshoot_Portfolio3.png" alt="snapshoot image project">
+            <div class="general">
+                <h3>Multi-Post Stories</h3>
+                <div id="general_inf_3">
+                     <p class="title">CANOPY</p>
+                     <img class="counter" src="images/Counter.png" alt="counter"/>
+                     <p class="job-title">Back End Dev</p>
+                     <img class="counter" src="images/Counter.png" alt="counter">
+                     <p class="year">2015</p>
+                </div>
+                <p class="project_description">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <footer>
+                  <ul id="skills_3" class="skills">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                  </ul>
+                  <div class="external_link">
+                    <a href="#">See Project</a>
+                  </div>
+                </footer>
+            </div>
+        </section>
+      </section>
+      <section class="final_wrap">
+        <section id="about_me">
+            <div class="data-inf">
+              <h1>About<br> Myself</h1>
+              <p id="presentation">I'm a front-end developer and I specialize in responsive web design. I can help you build a product, feature or website. Look through some of my work and experience!</p>
+              <div id="profile_link">
+                <h2>LET'S CONNECT</h2>
+                <ul>
+                    <li><a href="#"><img src="images/github.png" alt="github profilelink"></a></li>
+                    <li><a href="#"><img src="images/Linkedin_icon.png" alt="linkedin profile link"></a></li>
+                    <li><a href="#"><img src="images/doits.png" alt="doit"></a></li>
+                    <li><a href="#"><img src="images/twitter.png" alt="twitter profile link"></a></li>
+                    <li><a href="#"><img src="images/M.png" alt="M link"></a></li>
+                </ul>
+                <a id="my_resume" href="#">Get my resume</a>
+              </div>
+            </div>
+            <div id="my_skills">
+              <div id="languages_skills">
+                <div id="spec_name">
+                    <p>Languages</p>
+                    <div><img src="images/fleche_dessous.png" alt="selected"></div>
+                </div>
```

Comment:
Nice work on implementing the work section of this milestone, but some issues were missed in your design when aligning the items; please, as much as possible, stick to the Figma template you choose. :eagle: 

| Figma design | Your design |
|-------------------|------------------|
|![Screenshot from 2022-11-02 13-10-25](https://user-images.githubusercontent.com/65129228/199475754-72f425d4-6221-4a95-9fa1-64b6d983182c.png)| ![Screenshot from 2022-11-02 14-11-08](https://user-images.githubusercontent.com/65129228/199475977-6435ab8c-6012-4c69-b3eb-77d12db11c8b.png)|